wolfssl/wolfcrypt/ecc.h

```diff
@@ -462,13 +468,17 @@ struct ecc_key {
      * private D concatenated. These are respectively at offset `0`,
      * offset `keysize`, and offset `2 * keysize`.
      */
-    byte key_raw[3 * ECC_MAX_CRYPTO_HW_SIZE];
+    byte keyRaw[3 * ECC_MAX_CRYPTO_HW_SIZE];
```

Comment:
This will break some other ports like `silabs/silabs_ecc.c`. Please keep as `key_raw`.